src/test/store/profile-view.test.js

```diff
@@ -971,6 +971,55 @@ describe('actions/ProfileView', function() {
       expect(getMarker(markerIndexes[0]).name.includes('IPCOut')).toBeTruthy();
     });
 
+    it('filters the markers by a potential data payload of type Log', function() {
+      const profile = getProfileWithMarkers([
+        ['a', 0, null],
+        ['b', 1, null],
+        ['c', 2, null],
+        [
+          'd',
+          3,
+          {
+            type: 'Log',
+            module: 'nsJarProtocol',
+            name: 'nsJARChannel::nsJARChannel [this=0x87f1ec80]\n',
+          },
+        ],
+      ]);
+      const { dispatch, getState } = storeWithProfile(profile);
+
+      // Tests the filename
```

Comment:
I think these `// Tests...` comments would be better served as separated `it` blocks. The failures would then be much more granular.